presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -105,6 +105,17 @@ public void testCreateSampledTableAsSelectLimit()
     }
 
     @Test
+    public void testCreateTable()
```

Comment:
Does this need to be disabled for connectors that don't implement this? See `TestHiveDistributedQueries` or `TestCassandraDistributed` for an example